fedimint-client/src/lib.rs

```diff
@@ -1605,6 +1607,18 @@ pub async fn get_config_from_db(db: &Database) -> Option<ClientConfig> {
     config
 }
 
+pub async fn get_invite_code_from_db(db: &Database) -> Option<InviteCode> {
+    let mut dbtx = db.begin_transaction().await;
+    #[allow(clippy::let_and_return)]
+    let invite = dbtx
+        .find_by_prefix(&ClientInviteCodeKeyPrefix)
```

Comment:
**thought:** We should just have `DatabaseTransaction:find_singleton(key) -> Option<T>` with a default impl to make this pattern more usable.

Reply:
Thoughts @m1sterc001guy?